x-pack/plugins/security_solution/public/resolver/view/panels/node_events.tsx

```diff
@@ -94,6 +96,17 @@ const EventCategoryLinks = memo(function ({
         'data-test-subj': 'resolver:panel:node-events:event-type-count',
         width: '20%',
         sortable: true,
+        render(count: string) {
+          return (
+            <EuiToolTip position="bottom" content={numeral(count).format(defaultNumberFormat)}>
+              <EuiStat
```

Comment:
I like the idea of using `EuiStat` to format large values, but still seeing similar wrapping when `440826`, the same value from <https://github.com/elastic/kibana/issues/129987> is displayed, per the before / after comparison below:

![before_after_compare](https://user-images.githubusercontent.com/4459398/185214392-55e1ea50-ab71-49f5-8e15-29b1c5df67ad.png)

_Above: similar wrapping before / after with the value `440826`_

The `EuiStat` formatting of the number `440826` in the screenshot below is formatted as `440.826k`,  which feels unexpected:

![after](https://user-images.githubusercontent.com/4459398/185215834-9ee659c0-7aec-4e68-bc6c-217428212f35.png)

_Above: The value `440,826` in the tooltip is formatted as `440.826k` by `EuiStat`_

Consider:
- increasing the width of the `Count` column, because wrapping may still occur
- displaying the value using the formatted value from the tooltip instead of `EuiStat` if you agree that  `440.826k` feels like unexpected formatting